packages/flutter/lib/src/dom.dart

```diff
@@ -0,0 +1,350 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:js/js.dart';
+import 'package:js/js_util.dart' as js_util;
+
+/// This file includes static interop helpers for Flutter Web.
```

Comment:
I think this file should live in flutter/lib/src/services; also it should contain a comment that this will be eventually superseded by a `package:dom` or similar (and maybe create+link a Github issue where we track that we need to build that `package:dom`?)

Reply:
Moved, and created https://github.com/flutter/flutter/issues/113402